TrafficCapture/buildSrc/src/main/groovy/org/opensearch/migrations/common/CommonUtils.groovy

```diff
@@ -57,8 +57,8 @@ class CommonUtils {
                 runCommand("sed -i -e \"s|mirrorlist=|#mirrorlist=|g\" /etc/yum.repos.d/CentOS-* ;  sed -i -e \"s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g\" /etc/yum.repos.d/CentOS-*")
                 runCommand("yum -y install nmap-ncat")
             } else {
-                from 'openjdk:11-jre'
-                runCommand("apt-get update && apt-get install -y netcat")
```

Comment:
I don't believe that we need netcat any more.

Reply:
Agreed, I'd be surprised if we were using this anywhere and should probably remove

Reply:
Removed